packages/terra-tag/src/Tag.module.scss

```diff
@@ -39,7 +39,11 @@
     }
 
     &:focus {
-      outline: none;
+      border-color: var(--terra-tags-rollup-tag-focus-border-color, #868a8c);
+      border-width: var(--terra-tags-rollup-tag-focus-border-width, 1px);
+      box-shadow: var(--terra-tags-rollup-tag-focus-box-shadow, none);
+      outline: var(--terra-tags-rollup-tag-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-tags-rollup-tag-focus-outline-offset, 2px);
```

Comment:
Please add values for fusion and lowlight for relevant theme variables

Reply:
Changes Done. Thanks